DatadogSDKBridge/Classes/Implementation/DdRumImplementation.swift

```diff
@@ -137,4 +190,15 @@ internal class DdRumImplementation: DdRum {
         context[Self.timestampKey] = timestampMs
         return castAttributesToSwift(context)
     }
+
+    private func timingValue(from timings: [String: Any], for timingName: String) -> (start: Date, end: Date)? {
+        let timing = timings[timingName] as? [String: NSNumber]
+        if let startInNs = timing?["startTime"]?.int64Value, let durationInNs = timing?["duration"]?.int64Value {
+            return (
+                Date(timeIntervalSince1970: TimeInterval(Double(startInNs) / 1_000_000_000)),
+                Date(timeIntervalSince1970: TimeInterval(Double(startInNs + durationInNs) / 1_000_000_000))
```

Comment:
you may want to add a converter function to avoid having `1_000_000_000` around

```swift
(private?) extension TimeInterval {
  init(fromNs ns: Int64) { self = TimeInterval(ns) / 1_000_000 }
}
```

Reply:
thanks for the suggestion! implemented